segmenters/nlp/Sentencizer/__init__.py

```diff
@@ -32,13 +33,15 @@ def __init__(self,
                  max_sent_len: int = 512,
                  punct_chars: Optional[List[str]] = None,
                  uniform_weight: bool = True,
+                 lang: Optional[str] = 'en',
```

Comment:
Please remove the Optional annotation. It is only used if the default is `None`

```suggestion
                 lang: str = 'en',

```

Reply:
There should be no language, the sentrncizer looks for characters regardless of the language.

Reply:
okay, makes sense.
`string.printable` only gives the following characters: `'0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ!"#$%&\'()*+,-./:;<=>?@[\\]^_{|}~ \t\n\r\x0b\x0c'`
So your suggestion would be to remove the filter step completely?
I would agree.

Reply:
> Please remove the Optional annotation. It is only used if the default is `None`

It is used if the None option is properly handled, regardless of the default value.

Reply:
I think we should try to remove and see how it reacts to different inputs, like Chinese characters, arabic ones, and maybe `accent marks` and umlauds from different languages

Reply:
You are right